src/BIP44Node.test.ts

```diff
@@ -103,63 +66,17 @@ describe('BIP44Node', () => {
       });
     });
 
-    it('throws if the depth is invalid', async () => {
```

Comment:
Do we want to remove this test?

Reply:
`fromDerivationPath` derives the depth from the derivation path so there's no depth to verify.